app/helpers/submissions_helper.rb

```diff
@@ -51,6 +51,18 @@ def submission_solitary_author(submission)
     end.join(", ").presence || content_tag(:em, 'unknown author')
   end
 
+  def submission_solitary_author_forename(submission)
+    submission.students.map do |student|
+      account_forname(student)
+    end.join(", ").presence || content_tag(:em, 'unknown author')
```

Comment:
This should be a one-liner:

```suggestion
    submission.students.map(&:forename).keep_if(&:present?).join(", ").presence || content_tag(:em, 'unknown author')
```

Also, I am not sure if it is even possible that accounts are missing the `forename` and `surname` attribute. Please investigate. If my hunch is correct, you could do the following:

```suggestion
    submission.students.map(&:forename).join(", ").presence || content_tag(:em, 'unknown author')
```